ReactWindows/ReactNative/Bridge/FileJavaScriptBundleLoader.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+
+namespace ReactNative.Bridge
+{
+    class FileJavaScriptBundleLoader : IJavaScriptBundleLoader
+    {
+        private string _script;
+
+        public FileJavaScriptBundleLoader(string fileName)
+        {
+            SourceUrl = fileName;
+        }
+
+        public string SourceUrl { get; }
+
+        public async Task InitializeAsync()
+        {
+            try
+            {
+                var storageFile =
+                    await
+                        StorageFile.GetFileFromApplicationUriAsync(new Uri(SourceUrl))
+                            .AsTask()
+                            .ConfigureAwait(false);
+                _script = storageFile.Path;
+            }
+            catch (Exception ex)
+            {
+                var exceptionMessage = FormattableString.Invariant($"File read exception for asset '{SourceUrl}'.");
+                throw new InvalidOperationException(exceptionMessage, ex);
+            }
+        }
+
+        public void LoadScript(IReactBridge bridge)
+        {
+            if (bridge == null) throw new ArgumentNullException(nameof(bridge));
+
```

Comment:
Some had multi-line block braces, some didn't and I picked the most declarative style for this kind of boiler-plate. Happy to change it to whatever people feel most strongly for.